tests/README.md

```diff
@@ -0,0 +1,41 @@
+#Automation overview:
+
+SM-automation is written in Python3.7 and uses pytest Python package for running tests on cloud machines that are managed by GKE (google kubernetes engine).\
+Each test folder holds a test file (test_testname.py) and a config yaml file that contains test configurations, k8s configurations and SM-client arguments.
+
+Currently we use logs parsing in order to assert our tests but that will be changed to using the node's API.
+
+####Logical Test Objects Separation
+When starting a new test a namespace will be created for that test and that test only.\
+All deployments, statefulset and other objects will be created under that namespace and that namepsace only.
+
+####ELK Cluster
+Automation ELK is built from Fluent-bit, Logstash, Kibana and Elasticsearch.\
+We use fluent-bit as a shipper for the logs from SM-client to Logstash and modify those logs there before moving them to ES.\
+After setting the ELK cluster we wait for it to be ready, this process might take a couple of minutes.
+
+####Node pool
+Node pools provide a physical test objects separation in oppose to namespacing.\
+When ELK is ready we add the node pool that will serve the Spacemesh pods, this node pool is necessary in order to get all of the resources needed (only for the SM-clients) in advance so our deployment will happen in a shorter and more predictable time.\
+The process of creating\deleting\altering a node pool is blocking since only one action can be made simultaneously, this means that if someone just started a test and wants to create a node pool he will have to wait for previous started action to end.\
```

Comment:
can you remove all the \ ?
```suggestion
The process of creating\deleting\altering a node pool is blocking since only one action can be made simultaneously, this means that if someone just started a test and wants to create a node pool he will have to wait for previous started action to end.
```

Reply:
without those there will be no new line

Reply:
why do you have to have a new line?

Reply:
I don't see a reason to have a new line unless you break between different paragraphs, if it's the same paragraph, no new line is needed.

Reply:
@AmitShaul can you also replace the backslashes here ("creating\deleting\altering") with forward slashes ("/")?